src/robot.cpp

```diff
@@ -46,8 +46,8 @@ auto Robot::from(std::shared_ptr<moveit::core::RobotModel const> const& model,
         var.min = bounds.min_position_;
         var.max = bounds.max_position_;
 
-        var.clip_min = bounded ? var.min : -DBL_MAX;
-        var.clip_max = bounded ? var.max : +DBL_MAX;
+        var.clip_min = bounded ? var.min : std::numeric_limits<double>::lowest();
+        var.clip_max = bounded ? var.max : std::numeric_limits<double>::max();
```

Comment:
I do really like what you've done with this PR. These sentinel values here (and in the other places used in this repo) are things I think we should consider trying to represent in the interfaces of things as separate types. A lot of this is what I'd consider cruft held over from bio_ik. In many ways, double max is the nullptr of doubles.